lib/oidc/renewToken.ts

```diff
@@ -31,13 +31,27 @@ export function renewToken(sdk: OktaAuth, token: Token): Promise<Token> {
   }
 
   const { scopes, authorizeUrl, userinfoUrl, issuer } = token as (AccessToken & IDToken);
-  return getWithoutPrompt(sdk, {
-    responseType,
-    scopes,
-    authorizeUrl,
-    userinfoUrl,
-    issuer
-  })
+
+  // If we have a refresh token, renew using that, otherwise getWithoutPrompt
```

Comment:
I think it would make sense to add a helper renew function in TokenManager to make the flows check (it's actually looks like a `manager` function).

Also, as we expose the renew functions from the top-level (browser.ts), behavior change in the lower-level functions looks like a breaking change to me (changes the public API `authClient.token.renew`, which should use `getWithoutPrompt` only).

Reply:
Moved renew token with refresh token flow to TokenManager